src/commcare_cloud/ansible/deploy_prometheus.yml

```diff
@@ -0,0 +1,41 @@
+- name: Encrypt Data Drive
+  hosts: control1
+  become: true
+  roles:
+    - {role: ecryptfs, tags: 'ecryptfs'}
+
+- name: Prometheus
+  hosts: prometheus
+  become: true
+  tasks:
+    - include_role:
+        name: cloudalchemy.prometheus
+    - import_role:
+        name: dimagi.commcare_prometheus.prometheus
+  tags: prometheus
+
+- name: AlertManager
+  hosts: alertmanager
+  become: true
+  tasks:
+    - include_role:
+        name: cloudalchemy.alertmanager 
```

Comment:
any reason why this isn't listed in the dimagi role as a dependancy?

Reply:
Yeah it can be done